Makefile

```diff
@@ -1,7 +1,24 @@
-REGISTRY       ?= quay.io
-ORG            ?= kubernetes_incubator
-TAG            ?= $(shell git describe --always --dirty)
-IMAGE          ?= $(REGISTRY)/$(ORG)/node-feature-discovery-operator:$(TAG)
+.PHONY: all test
+.FORCE:
+
+GO_CMD := go
+GO_FMT := gofmt
+
+IMAGE_BUILD_CMD ?= docker build
+IMAGE_BUILD_EXTRA_OPTS :=
```

Comment:
Could use `?=` for all "customization variables" in the way you proposed for NFD. WDYT(?)

I think that would include all variables from here to `IMAGE_EXTRA_TAGS`, excluding `VERSION`, `IMAGE_NAME` and possibly `IMAGE_REPO`.